src/main/java/edu/cornell/kfs/concur/service/impl/ConcurEventNotificationConversionServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package edu.cornell.kfs.concur.service.impl;
+
+import java.text.ParseException;
+
+import org.kuali.rice.core.api.datetime.DateTimeService;
+
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotification;
+import edu.cornell.kfs.concur.rest.xmlObjects.ConcurEventNotificationDTO;
+import edu.cornell.kfs.concur.service.ConcurEventNotificationConversionService;
+
+public class ConcurEventNotificationConversionServiceImpl implements ConcurEventNotificationConversionService {
+    static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurEventNotificationConversionServiceImpl.class);
+    
+    protected DateTimeService dateTimeService;
+
+    @Override
+    public ConcurEventNotification convertConcurEventNotification(ConcurEventNotificationDTO concurEventNotificationDTO) {
+        ConcurEventNotification concurEventNotification = new ConcurEventNotification();
+        concurEventNotification.setContext(concurEventNotificationDTO.getContext());
+        try {
+            concurEventNotification.setEventDateTime(dateTimeService.convertToSqlDate(concurEventNotificationDTO.getEventDateTime()));
+        } catch (ParseException e) {
+            LOG.warn("Incoming EventDateTime was invalid and it was reset to the current date");
+            concurEventNotification.setEventDateTime(dateTimeService.getCurrentSqlDate());
```

Comment:
I think this is fine for now, but we should talk about this logic a little more and make sure it is really what we want. If not, we can change it when working on a future story.